restapi.go

```diff
@@ -472,6 +472,29 @@ func (s *Session) Guild(guildID string) (st *Guild, err error) {
 	return
 }
 
+// GuildWithCounts returns a Guild structure of a specific Guild.
+// guildID    : The ID of a Guild
+// withCounts : When true, will return approximate member and presence counts for the guild
+//              uses variadic for backwards compatibility
+func (s *Session) GuildWithCounts(guildID string, withCounts ...bool) (st *Guild, err error) {
```

Comment:
Since this is a separate function now, I think we can drop variadic arguments here.